<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Favoritos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/valknut.ico</normaloff>../img/valknut.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form{
background-color: #000000;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="sclFavoritos">
     <property name="styleSheet">
      <string notr="true">QScrollArea#sclFavoritos{
background-color: rgb(20,20,20);
border-style: solid;
border-width: 1px;
border-color: #7d0304;
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scpFavoritos">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>343</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#scpFavoritos{
background-color: rgb(20,20,20);
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="2" column="3">
        <widget class="QPushButton" name="btnFavInserir">
         <property name="styleSheet">
          <string notr="true">QPushButton#btnFavInserir{
color: #FFFFFF;
background-color: rgb(0,0,0);
border-style: solid;
border-width: 1px;
border-color: #7d0304;
padding-top: 5px;
padding-bottom: 5px;
cursor: default;
}
QPushButton#btnFavInserir:hover{
background-color: rgb(30,30,30);
cursor: hand;
}
QPushButton#btnFavInserir:pressed{
background-color: rgb(20,20,20);
cursor: pointer;
}</string>
         </property>
         <property name="text">
          <string>Inserir</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnFavDeletar">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#btnFavDeletar{
color: #FFFFFF;
background-color: rgb(0,0,0);
border-style: solid;
border-width: 1px;
border-color: #7d0304;
padding-top: 5px;
padding-bottom: 5px;
cursor: default;
}
QPushButton#btnFavDeletar:hover{
background-color: rgb(30,30,30);
cursor: hand;
}
QPushButton#btnFavDeletar:pressed{
background-color: rgb(20,20,20);
cursor: pointer;
}</string>
         </property>
         <property name="text">
          <string>Deletar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnFavDeletarTodos">
         <property name="styleSheet">
          <string notr="true">QPushButton#btnFavDeletarTodos{
color: #FFFFFF;
background-color: rgb(0,0,0);
border-style: solid;
border-width: 1px;
border-color: #7d0304;
padding-top: 5px;
padding-bottom: 5px;
cursor: default;
}
QPushButton#btnFavDeletarTodos:hover{
background-color: rgb(30,30,30);
cursor: hand;
}
QPushButton#btnFavDeletarTodos:pressed{
background-color: rgb(20,20,20);
cursor: pointer;
}</string>
         </property>
         <property name="text">
          <string>Deletar Todos</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btnFavAlterar">
         <property name="styleSheet">
          <string notr="true">QPushButton#btnFavAlterar{
color: #FFFFFF;
background-color: rgb(0,0,0);
border-style: solid;
border-width: 1px;
border-color: #7d0304;
padding-top: 5px;
padding-bottom: 5px;
cursor: default;
}
QPushButton#btnFavAlterar:hover{
background-color: rgb(30,30,30);
cursor: hand;
}
QPushButton#btnFavAlterar:pressed{
background-color: rgb(20,20,20);
cursor: pointer;
}</string>
         </property>
         <property name="text">
          <string>Alterar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="btnFavIr">
         <property name="styleSheet">
          <string notr="true">QPushButton#btnFavIr{
color: #FFFFFF;
background-color: rgb(0,0,0);
border-style: solid;
border-width: 1px;
border-color: #7d0304;
padding-top: 5px;
padding-bottom: 5px;
cursor: default;
}
QPushButton#btnFavIr:hover{
background-color: rgb(30,30,30);
cursor: hand;
}
QPushButton#btnFavIr:pressed{
background-color: rgb(20,20,20);
cursor: pointer;
}</string>
         </property>
         <property name="text">
          <string>Ir</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QTableWidget" name="tblFavoritos">
         <property name="styleSheet">
          <string notr="true">color: black;
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QLineEdit" name="txtFavUrl">
         <property name="styleSheet">
          <string notr="true">QLineEdit#txtFavUrl{
color: black;
background-color: rgb(255,255,255);
border-style: solid;
border-width: 1px;
border-color: #7d0304;
padding-top: 5px;
padding-bottom: 5px;
cursor: default;
}</string>
         </property>
         <property name="placeholderText">
          <string>Digite aqui a Url</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
